WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -26,7 +26,7 @@
       <Setter Property="Background" Value="Transparent" />
       <Setter Property="Height" Value="24" />
     </Style>
-    <Style Selector="Grid.CoinItemGrid > :is(Control)">
+    <Style Selector="Grid.CoinItemGrid > :is(Control):not(:is(Control).FocusTarget)">
```

Comment:
If you set the horizontalalignment locally on the FocusTarget panel, this style wont be able to override it, and this complex selector will not be needed... remember there is a performance hit for styles like this.